portfolio/src/main/webapp/index.html

```diff
@@ -194,9 +194,15 @@ <h2>NumC</h2>
     </section>
 
     <section class="contact" id="contact">
-      <div class="heading wblack">
+      <div class="heading black">
```

Comment:
If you have a single "heading" class that doesn't include the color then you can switch the color in the css later without needing to push html changes.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation